state/watchlist/operations/fetchWatchlist.js

```diff
@@ -0,0 +1,33 @@
+import { createLogic } from 'redux-logic';
+
+import { FETCH_WATCHLIST } from '../types';
+import normalizedMovies from '../../dashboard/normalize/movies';
```

Comment:
It's better to move this file to some shared folder because you are using it in different operations but it's placed in `dashboard`.

Reply:
Done